File: 12-title-case/title-case.js
// create a function that capitalizes the first letter of every word in a given string.

const titleCase=(str)=>{

    const strArry=str.split(' ')

    const result=[]
    
    for(var i=0; i<strArry.length; i++){
        
        const firstLetter=strArry[i][0].toUpperCase()
        const restOfWord=strArry[i].substring(1, strArry[i].length)
        result.push(firstLetter+restOfWord)
    }

    return result.join(' ')
}

console.log(titleCase('a lannister always pays his debts'))

File: 12-title-case/title_case.py
def title_case(string):

    string_list=string.split(' ')
    
    result=[]
    for word in string_list:
        first_letter=word[0].upper()
        rest_of_word=word[1:len(word)]

        result.append(first_letter+rest_of_word)

    return ' '.join(result)
        


print(title_case('the quick brown fox jumped over the lazy dog'))